spec/aggregates/reviewing/review_spec.rb

```diff
@@ -36,8 +38,8 @@
     let(:user_id) { SecureRandom.uuid }
 
     before do
-      review.receive_application(application_id:)
-      review.send_back(application_id:, user_id:, reason:)
+      review.receive_application(submitted_at:)
+      review.send_back(user_id:, reason:)
     end
 
     it 'is becomes "sent_back"' do
```

Comment:
Not changed in this PR, but the grammar is wrong here "it is becomes "sent_back" --> it becomes "sent_back"